packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/agent/volumes.py

```diff
@@ -0,0 +1,54 @@
+import logging
+from datetime import timedelta
+
+from models_library.projects_nodes_io import NodeID
+from models_library.rabbitmq_basic_types import RPCMethodName, RPCNamespace
+from pydantic import parse_obj_as
+from servicelib.logging_utils import log_decorator
+from servicelib.rabbitmq import RabbitMQRPCClient
+
+_logger = logging.getLogger(__name__)
+
+
+@log_decorator(_logger, level=logging.DEBUG)
+async def remove_volumes_without_backup_for_service(
+    rabbitmq_rpc_client: RabbitMQRPCClient,
+    *,
+    docker_node_id: str,
+    swarm_stack_name: str,
+    node_id: NodeID,
+) -> None:
+    result = await rabbitmq_rpc_client.request(
+        RPCNamespace.from_entries(
+            {
+                "service": "agent",
+                "docker_node_id": docker_node_id,
+                "swarm_stack_name": swarm_stack_name,
+            }
+        ),
+        parse_obj_as(RPCMethodName, "remove_volumes_without_backup_for_service"),
+        node_id=node_id,
+        timeout_s=int(timedelta(minutes=60).total_seconds()),
```

Comment:
this is an interesting way of computing one minute in seconds every single time the call is done :-) 
Even if you want to keep the "algorithm", perhaps you can add it as a constant at the module level ?